.github/ISSUE_TEMPLATE/30-feature-request.md

```diff
@@ -0,0 +1,35 @@
+---	
+name: Feature Request - If you have something new you would like to add	
+about: Please follow the given template	
+title: ''	
+labels: 'Feature: Request'	
+assignees: ''	
+
+---	
+
+<!-- PLEASE DO NOT DELETE TOPICS AS YOUR ISSUE WILL GET CLOSED -->
+### Prerequisites <!-- Replace spaces with "x" to tick boxes. -->
+
+- [ ] I am running the latest **alpha** version of Structurize for my Minecraft version.
+- [ ] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
+- [ ] I made sure that **this feature request is not a duplicate of any existing feature request.**
```

Comment:
We aren't going to have the "this is a feature request and not a bug" checkbox anymore?
Also just nit-pick suggestion "made sure my issue is not covered there" -> "made sure the feature doesn't already exist"

(same for change request of course)

Reply:
Look at config.yml, you will see them once merged along with issue templates

Reply:
Yep, just thought we'd make sure 'cause I reckon people will miss the links (though tbf they'd probably ignore the checkbox as well if the automation will let them get away with it). All good.